src/odata-to-abstract-sql.ts

```diff
@@ -1469,17 +1469,16 @@ const generateShortAliases = (clientModel: AbstractSqlModel) => {
 
 	const getRelationships = (
 		relationships: AbstractSqlModel['relationships'] | Relationship,
+		nestedRelationships: string[] = []
```

Comment:
Should we also add a comment that this is an `out` param and shouldn't be used as an arg? (Sorry for the C# terminology)